cmd/controller-manager/app/helper.go

```diff
@@ -53,3 +54,26 @@ func WaitForAPIServer(client clientset.Interface, timeout time.Duration) error {
 
 	return nil
 }
+
+// IsControllerEnabled check if a specified controller enabled or not.
+func IsControllerEnabled(name string, disabledByDefaultControllers sets.String, controllers ...string) bool {
```

Comment:
the signature is slightly odd: why a ... variable for controllers? It's not a variadic usage here.

Better:
```
func IsControllerEnabled(name string, all, disabledByDefault sets.String) bool {
```

Reply:
or `func IsControllerEnabled(name string, all []string, disabledByDefault sets.String) bool {` if that fits better.

Reply:
accept